middleware/authorization.js

```diff
@@ -10,6 +10,9 @@ function verifyRole(roleName) {
       if (!userId) {
         return respondWithError(res, 401, "User not authenticated");
       }
+      // Check if user is admin first
+      const isAdmin = await repo.userIsAdmin(userId);
+      if (isAdmin) return next();
```

Comment:
let's add some log line before silently bypassing other checks, log user id and (?) the route